assets/global.js

```diff
@@ -701,6 +701,7 @@ class SlideshowComponent extends SliderComponent {
     super();
     this.sliderControlWrapper = this.querySelector('.slider-buttons');
     this.enableSliderLooping = true;
+    this.arrowButtonWasInteracted = false;
```

Comment:
I'm kind of wondering if we should wrap some of that code in a conditional. 

Something like: 
```javascript
if (this.classList.contains('announcement-bar') {
  ... run the logic you're adding
```
Or create a variable that is `this.announcementBarSlideshow = this.classList.contains('announcement-bar')`

Reply:
I think that kind of condition I already have in line 744 - 751, but ideally we want to separate all additional logic into a new component or extension for the slideshow component. Let's do it as a next step.

Reply:
[it's done](https://github.com/Shopify/dawn/pull/2580/commits/00d23e0bf98fe7beb6aa47d33c5b1e02699bd14c#diff-47a8b2f1172e03da15783b379f0e007d8e3dfd27cbd9142adf9e32c4b9591079R715-R728).